### PR TITLE
Inlined and minified grayscale.js, also optimized

### PR DESCRIPTION
**Pull request by njaremko**
Also, optimized all the images that the site uses by changing png's to 8 bit and optimizing IDAT stream.

### CODE REVIEW

**Comment by PanosSakkos** [comment]
Awesome! 

I'm abroad for the weekend, I will merge it on Sunday, sorry for the delay :smile:

**Comment by njaremko** [comment]
In case anyone's curious, the image down-scaling (they were way bigger than necessary) and optimization made the img folder 5.74x smaller. (So make sure you're optimizing you images on your sites)

**Comment by joariasl** [comment]
Is there some form of minified JS in build time Jekyll?
The problem is that the code will not be easily compressible.

**Comment by njaremko** [comment]
Well, it's open source easy to find bootstrap code, so that's not a big
deal.

Also, we probably won't ever be changing it.

I'm regard to build time js minifying, Google's closure js compiler has an
api that we might be able to call, haven't really looked into it...but the
api call probably isn't worth it if we can just inline it by default.

(Have said that, the api might be worth looking at for the typed js)

**Comment by PanosSakkos** [comment]
@joariasl I had the same concern but I found this http://unminify.com/ :smile:

File: _includes/js.html [hunk 1]
```diff
@@ -11,7 +11,11 @@
 <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-easing/1.3/jquery.easing.min.js"></script>
 
 <!-- Custom Theme JavaScript -->
-<script src="{{site.baseurl}}/js/grayscale.js"></script>
+<!--* Start Bootstrap - Grayscale Bootstrap Theme (http://startbootstrap.com)
+* Code licensed under the Apache License v2.0.
+* For details, see http://www.apache.org/licenses/LICENSE-2.0.-->
+<script>$(window).scroll(function(){50<$(".navbar").offset().top?$(".navbar-fixed-top").addClass("top-nav-collapse"):$(".navbar-fixed-top").removeClass("top-nav-collapse")});$(function(){$("a.page-scroll").bind("click",function(b){var a=$(this);$("html, body").stop().animate({scrollTop:$(a.attr("href")).offset().top},1500,"easeInOutExpo",function(){a.blur()});b.preventDefault()})});$(".navbar-collapse ul li
+a").click(function(){$(".navbar-toggle:visible").click()});</script>
```
Comment (PanosSakkos) [inline _includes/js.html]: This new line breaks the code :sweat_smile:

Comment (PanosSakkos) [inline _includes/js.html]: This new line breaks the code :sweat_smile:

**Comment by njaremko** [comment]
Something must have gone wrong in my copy pasting of the minified code :sweat_smile:

Should be good now :+1:

**Comment by PanosSakkos** [comment]
No worries! :smile:

**Pull request merged**